### ISSUE TITLE
Add an overlay to the game cover based on it's Library

### ISSUE THREAD

**Issue by RubeensVinicius** [JosefNemec/Playnite#1450]
Recently I was looking for covers for my games on Reddit and found [this post](https://www.reddit.com/r/steamgrid/comments/d0c0jy/o_my_custom_overlays_albums_in_comments/) where the user made an overlay for each console/launcher that could be added to the grid image with an image editor before adding it to Steam. That's a great idea to make the same game on different launchers more distinct on Playnite, but doing this manually would be a drag.

An option to associate each Library with an overlay image to be layered on the game's cover on the grid view would be really nice.

**Comment by JosefNemec** [comment]
I'm not sure about creating another option for overlay image, but we could technically display platform or library icon in that way.

**Comment by jameskchapman-github** [comment]
@JosefNemec @RubeensVinicius I posted something very similar to this on r/Playnite before I knew of these forums

https://www.reddit.com/r/playnite/comments/cmsicr/would_be_awesome_if_we_could_add_a_banner_to_each/

**Comment by shakeyourbunny** [comment]
I'd suggest transforming the specific overlay icons to a set of say 3 overlay icons / badges where one can not only select completion status or some platform pictures, but also for tags and or genres.

**Comment by GinoMan** [comment]
I'm thinking about it and it would be cool to have a set of icons I could overlay that indicate if you should use a controller and what kind, like an icon for DS4, one for Keyboard and Mouse, one for Wiimote, One for Wiimote and Nunchuck, one for GameCube Controllers, and one for XBox360/XBox One. Overlay Icons would be the least intrusive IMO. 

Maybe something like these: 

![Overlay Icons](https://user-images.githubusercontent.com/528834/93551472-f3639d80-f93b-11ea-9180-0061d41e0852.png)

In action:

![Ace Combat - Air Combat - Sample](https://user-images.githubusercontent.com/528834/93552725-468b1f80-f93f-11ea-80eb-3a7fcacb4fea.png)

![Alwa's Awakening - Sample](https://user-images.githubusercontent.com/528834/93552733-4c810080-f93f-11ea-8adf-86c755208a30.png)

![Brutal Doom - Sample](https://user-images.githubusercontent.com/528834/93552738-50148780-f93f-11ea-9578-bf1346001473.png)

![Metroid Prime Trilogy - Sample](https://user-images.githubusercontent.com/528834/93552741-5440a500-f93f-11ea-8fcc-470041b417ef.png)

![Metroid Samus Returns - Sample](https://user-images.githubusercontent.com/528834/93552751-57d42c00-f93f-11ea-964c-f4bd8a64b257.png)

![Sonic and the Secret Rings - Sample](https://user-images.githubusercontent.com/528834/93552762-5f93d080-f93f-11ea-9d4f-18a570d5a5d6.png)

![The Legend of Zelda - Wind Waker - Sample](https://user-images.githubusercontent.com/528834/93552765-64f11b00-f93f-11ea-9176-b74ab7787eac.png)

I think this would allow someone to see visually what kind of controller is needed for any particular game, whether emulated or on PC.

**Comment by teconmoon** [comment]
I realize this is an old feature request but this is exactly what I came to GitHub to request. I primarily use Playnite in Fullscreen mode + NVIDIA GameStream, so being able to show an icon for what's required to play a game for family members who aren't familiar with what's needed for a specific game would be super neat.

**Comment by ccrowles** [comment]
I still wish this would get implemented, every time I have to manually edit game covers in my library just to add the platform.

Example of a GamePass "ribbon" I found and use to mark gamepass games:

![f3150bdaf9209ce3a24c74c207dba87d](https://github.com/JosefNemec/Playnite/assets/12866474/dfa807cf-0e5f-4d2f-95ed-2067aeb88e7d)

**Comment by UnionExxis** [comment]
You can do it with some custom themes, but I agree, since there are so many different platforms that you can have in Playnite, it would be nice to have an ability to add a ribbon (like in the image you showed) in the default program.

**Comment by ccrowles** [comment]
There was a [nice implementation of it for Steam itself](https://github.com/boppreh/steamgrid) that someone used to [make ribbons for the SteamDeck](https://new.reddit.com/r/steamgrid/comments/wo5k8s/overlays_for_steamdeck/), and I really wish we had something similar for Playnite. Especially because otherwise game demos are impossible to distinguish from full games unless you click on the game itself and read the name.

**Comment by hispanico957** [comment]
PLS some new regarding this features: Option to add picture above covers for games (according to platform)
I search on post but i dont find anything if is possible or not 
Refer  https://github.com/JosefNemec/Playnite/issues/1815
Thank

Kira

**Comment by ccrowles** [comment]
> You can do it with some custom themes, but I agree, since there are so many different platforms that you can have in Playnite, it would be nice to have an ability to add a ribbon (like in the image you showed) in the default program.

Yeah, that's why I showed the ribbon example. Since it's just a ribbon, it doesn't take any extra space pixel-wise. Custom themes like eMixedNite (which is the theme i use) on the other hand take extra pixels because they add an extra banner above and/or below the actual game cover.
A ribbon would be the most easily identifiable and non-invasive option.

**Comment by hispanico957** [comment]
> > You can do it with some custom themes, but I agree, since there are so many different platforms that you can have in Playnite, it would be nice to have an ability to add a ribbon (like in the image you showed) in the default program.
> 
> Yeah, that's why I showed the ribbon example. Since it's just a ribbon, it doesn't take any extra space pixel-wise. Custom themes like eMixedNite (which is the theme i use) on the other hand take extra pixels because they add an extra banner above and/or below the actual game cover. A ribbon would be the most easily identifiable and non-invasive option.

YESSSSSS thank but in which way i can add this ribbon on my covers games.
Thank
Kira

**Comment by ccrowles** [comment]
> YESSSSSS thank but in which way i can add this ribbon on my covers games.

I just either search for them or make them on photoshop, then manually overlay them on the cover (again on photoshop) and then manually change the cover image. It's tedious and it took me weeks to do it for my whole library, but it works.

**Comment by hispanico957** [comment]
I find and solved with this beautiful theme Theme-KNARZnite
https://github.com/HerrKnarz/Playnite-Theme-KNARZnite?tab=readme-ov-file

Hispa

**Comment by RetroFlock** [comment]
I personally use the Seaside theme, but that's just me
